code/modules/clothing/spacesuits/rig/suits/combat.dm

```diff
@@ -332,3 +332,79 @@
 
 /obj/item/clothing/head/helmet/space/rig/combat/solfed
 	camera_networks = null//list(NETWORK_SOYFED) //It is over general, I have depicted your murderous federation as the soyjack and my glorious homeland as the chad. To do, maybe actually set this up.
+
+//Hunters
+
+/obj/item/rig/combat/hunter
+	name = "Lodge \"Apex\" control module"
+	desc = "A protective suit made by a local hunting lodge, the raw material for which was the most terrible creatures of this world.\
+	 A technological miracle fueled by the power of oddity fused with the savvy of the hunting craft.\
+	 The wearer of this costume receives the powers of the enemies he defeated! So who's the apex predator here now?!"
```

Comment:
Costume is an interesting term to use here- it may not be appropriate since a costume is definitionally a fake thing you wear to pretend to be something